src/actor/Claimer.ts

```diff
@@ -77,6 +79,12 @@ export async function createPresentation(
     if (!noNulls(peCreds)) {
       throw ERROR_PE_CREDENTIAL_MISSING()
     }
+    if (!identity.claimer) {
+      throw ERROR_IDENTITY_NOT_PE_ENABLED()
+    }
+    if (!request.content.peRequest) {
+      throw ERROR_PE_MISSING()
```

Comment:
Currently, we are not testing this throw. Maybe we should.

Reply:
fixed in ea38166a4ce1b9c442db0264098e7f7ff080847f